test/cfa/ntp_conf_test.rb

```diff
@@ -287,19 +369,23 @@ def ntp_disk_file
       options = ["default", "notrap"]
       record.options = options
       expect(record.options).to eq(options)
+      ntp.save
+      expect(file.content).to include("restrict -4 default notrap\n")
     end
   end
 
   describe "#raw_options" do
     it "obtains options as string" do
-      expect(record.raw_options).to eq("default")
+      expect(record.raw_options).to eq("default notrap nomodify nopeer")
     end
   end
 
   describe "#raw_options=" do
     it "sets options from a string" do
       record.raw_options = "default notrap"
       expect(record.options).to eq(["default", "notrap"])
+      ntp.save
+      expect(file.content).to include("restrict -4 default notrap\n")
```

Comment:
Right now it is not possible because it entails several changes. Augeas structure is sligtly different for these two cases. Anyway, ntp-client does not allow to change restrict entries currently.